src/components/MultiHopTrade/components/LimitOrderV2/LimitOrderConfirm.tsx

```diff
@@ -0,0 +1,405 @@
+          .times(feeAssetRateUserCurrency)
+          .toFixed()
+        return (
+          <Stack spacing={4} width='full'>
+            <Row>
+              <Row.Label>
+                <Text translation='limitOrder.networkFee' />
+              </Row.Label>
+              <Row.Value>
+                <Skeleton isLoaded={!isLoadingAllowanceReset}>
+                  <HStack justifyContent='flex-end'>
+                    <Amount.Crypto
+                      value={allowanceResetNetworkFeeCryptoPrecision}
+                      symbol={feeAsset?.symbol ?? ''}
+                    />
+                    <Amount.Fiat
+                      color={'text.subtle'}
+                      prefix='('
+                      suffix=')'
+                      noSpace
+                      value={allowanceResetNetworkFeeUserCurrency}
+                    />
+                  </HStack>
+                </Skeleton>
+              </Row.Value>
+            </Row>
+          </Stack>
+        )
+      case LimitOrderSubmissionState.AwaitingLimitOrderSubmission:
+        return <LimitOrderDetail />
+      default:
+        return null
+    }
+  }, [
+    allowanceResetNetworkFeeCryptoBaseUnit,
+    approvalNetworkFeeCryptoBaseUnit,
+    feeAsset?.precision,
+    feeAsset?.symbol,
+    feeAssetRateUserCurrency,
+    isLoadingAllowanceApproval,
+    isLoadingAllowanceReset,
+    orderSubmissionState,
+  ])
+
+  const buttonTranslation: string | [string, number | InterpolationOptions] | undefined =
+    useMemo(() => {
+      if (!isConnected || isDemoWallet) return 'common.connectWallet'
+      switch (orderSubmissionState) {
+        case LimitOrderSubmissionState.AwaitingAllowanceApproval:
+          return ['trade.approveAsset', { symbol: sellAsset?.symbol }]
+        case LimitOrderSubmissionState.AwaitingAllowanceReset:
+          return 'trade.resetAllowance'
+        case LimitOrderSubmissionState.AwaitingLimitOrderSubmission:
+          return 'limitOrder.placeOrder'
+        default:
+          return undefined
+      }
+    }, [isConnected, isDemoWallet, orderSubmissionState, sellAsset?.symbol])
+
+  const handleConfirm = useCallback(async () => {
+    if (!isConnected || isDemoWallet) {
+      walletDispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+      return
+    }
+    if (!quoteId) {
+      console.error('Attempting to confirm with undefined quoteId')
+      return
+    }
+    switch (orderSubmissionState) {
+      case LimitOrderSubmissionState.AwaitingAllowanceApproval:
+        allowanceApprovalMutation.mutate()
+        break
+      case LimitOrderSubmissionState.AwaitingAllowanceReset:
+        allowanceResetMutation.mutate()
+        break
+      case LimitOrderSubmissionState.AwaitingLimitOrderSubmission:
+        const result = await placeLimitOrder({ quoteId, wallet })
+
+        // Exit if the request failed.
+        if ((result as { error: unknown }).error || !!result) {
```

Comment:
blocking: found her! 🐛 

Confirmed we're now gucci after

https://jam.dev/c/3f46a2bc-5669-4bab-bcbd-317cd564babd

```suggestion
        if ((result as { error: unknown }).error || !result) {
```

Reply:
Doh! Good find, I fat-thumbed the exclamation in my last couple of commits to get this tidied up for review 🤦‍♂️

Reply:
https://github.com/shapeshift/web/pull/8608/commits/256e416309466c17495f0d030b1c87591a776bd0

Reply:
And this one is now happy when happy! https://jam.dev/c/d764272a-bf53-4886-a54d-a3ae45876c43